glide.lock

```diff
@@ -383,7 +383,7 @@ imports:
   subpackages:
   - log
 - name: github.com/emicklei/go-restful-swagger12
-  version: 5e28dc7157ebd9247a268c26dd1bee7818aa33f7
+  version: f1854a45c13b3f2b1e6df2f01554c638c1621df9
```

Comment:
this isn't right: https://github.com/openshift/emicklei-go-restful-swagger12/commit/f1854a45c13b3f2b1e6df2f01554c638c1621df9

Reply:
no. your bump is referencing a commit that doesn't exist.